src/main/java/Main.java

```diff
@@ -0,0 +1,133 @@
+import java.util.Scanner;
+import java.util.ArrayList;
+
+public class Main {
+    Scanner sc=new Scanner(System.in);
+    private final String myName;
+    ArrayList<Task> tasks;
+
+    public Main(String myName) {
+        this.myName = myName;
+        tasks=new ArrayList<>();
+    }
+
+    public void printHorizontalLine() {
+        System.out.println("______________________________________________");
+    }
+
+    public boolean isValidTaskNumber(int taskNumber) {
+        return (taskNumber >= 1 && taskNumber <= tasks.size());
+    }
+
+
+    public void giveIntroduction() {
+        System.out.println("Hello I am " + myName + ".");
+        System.out.println("What can I do for you?");
+        printHorizontalLine();
+    }
+
+    public void sayGoodbye() {
+        System.out.println("Bye. Hope to see you again soon!");
+        printHorizontalLine();
+    }
+
+    public void listTasks() {
+        System.out.println("Here are the tasks in your list: ");
+        int serialNumber=0;
+        for(Task task : tasks) {
+            serialNumber++;
+            System.out.println(serialNumber + "." + task.toString());
+        }
+        printHorizontalLine();
+    }
+
+
+    public void changeTaskStatus(String enteredString) {
+        int taskNumber= Integer.parseInt(enteredString.substring(enteredString.length()-1));
```

Comment:
Perhaps can break it down abit instead of all in a line to allow more readability